src/features/web-compat.js

```diff
@@ -72,6 +72,29 @@ function permissionsFix (settings) {
     }
 }
 
+function messageHandlersFix () {
+    // Do nothing if `messageHandlers` is absent
+    if (!window.webkit?.messageHandlers) return
+
+    // handlers in any page-world user-scripts
+    const knownHandlers = ['trackerDetectedMessage', 'printHandler', 'specialPages']
+
+    window.webkit = {
+        messageHandlers: new Proxy(window.webkit.messageHandlers, {
```

Comment:
Should this be DDGProxy instead? or defineProperty or another of the wrappers. These handle toString and now debug flags.

Reply:
I couldn't get DDGProxy to work - but I'll try the others